templates/zerver/emails/notify_new_login.html

```diff
@@ -15,3 +19,10 @@
 <p>If you recognize this login activity, you can archive this notice.</p>
 <p>Thanks,<br />Zulip Account Security</p>
 {% endblock %}
+
+{% block manage_preferences %}
```

Comment:
We probably shouldn't put an unsubscribe link here since this seems like the kind of thing that should be mandatory to send/receive, or at least we probably want this to be an "unsubscribe from new login notifications" type link/messaging if we allow unsubscribing.